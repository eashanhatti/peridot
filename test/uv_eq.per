def x: Struct {} = fun(x, y) => struct { foo = x }
