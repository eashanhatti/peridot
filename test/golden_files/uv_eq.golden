((fromList [],
  fromList [],
  fromList
    [(SourcePos
        {sourceName = "test//uv_eq.per",
         sourceLine = Pos 1,
         sourceColumn = Pos 20},
      InferredFunType (RecType (fromList [])))]),
 Declare
   Obj
   (Rigid
      (RNameIntro (UserName "x") Obj (Id {unId = 0})))
   (RecType (fromList []))
   (Define
      (Rigid
         (RNameIntro (UserName "x") Obj (Id {unId = 0})))
      (Rigid ElabError)
      (Rigid (TypeType Obj))))
